models/attack.go

```diff
@@ -43,3 +47,45 @@ func StatusFilter(status string) Filter {
 		return false
 	}
 }
+
+// CreationBeforeFilter implements an attack created_before filter
+// in the Filter function format
+func CreationBeforeFilter(d string) Filter {
+	return func(a AttackDetails) bool {
+		if d == "" {
+			return true
+		}
+		const layoutUser = "2006-01-02"
```

Comment:
Wondering if it might be better to take it down to the hour of the day ?! Thinking more from the front-end experience.

Reply:
Consider it as done !